### ISSUE TITLE
Math formulas show scrollbars on Windows

### ISSUE THREAD

**Issue by wbwqq** [squidfunk/mkdocs-material#2048]
## Description

Block math rendering is not displaying correctly. The block math statements are full of scrollbars.

![Screenshot_2020](https://user-images.githubusercontent.com/74458558/99188927-e22de600-275e-11eb-9a63-cfb6695b2454.png)

### Expected behavior

Math rending should be without any scrollbars.

### Actual behavior

Scrollbars all round block math rendering.

### Steps to reproduce the bug

With either defaults (adding `extra_javascript:...` to `mkdocs.yml` or adding `config.js`) for the inclusion of MaxJax config and JavaScript runtime from the docs, the problem persists.

### Package versions

* Python: 3.8.5
* MkDocs: 1.1.2
* Material: 6.1.4
* Markdown 3.3.3

### Project configuration

``` yaml
site_name: something notes
theme:
    name: material
    palette:
        scheme: slate
markdown_extensions:
    - admonition
    - pymdownx.highlight
    - pymdownx.inlinehilite
    - pymdownx.superfences
    - pymdownx.keys
    - meta
    - pymdownx.arithmatex:
        generic: true

extra_javascript:
  - javascripts/config.js
  - https://polyfill.io/v3/polyfill.min.js?features=es6
  - https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js

```

### System information

* OS: win10
* Browser: firefox 82.0.3

  [1]: https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md

**Update** Added Markdown version

**Comment by squidfunk** [comment]
Thanks for the bug report. Could you please provide the Markdown you used? It's easier for us to reproduce. Unfortunately, I don't have any Windows machine, so maybe somebody with a Windows machine could try to debug this 😅

**Comment by wbwqq** [comment]
I added the Markdown version (3.3.3) in an edit.

**Comment by squidfunk** [comment]
> I added the Markdown version (3.3.3) in an edit.

The Markdown content you used for the formula, i.e. the content of your `*.md` file.

**Comment by squidfunk** [comment]
Okay, so the only obvious thing I found was that `.arithmatex` blocks had `overflow: scroll` set, which will always show a scrollbar on Windows. This was fixed by 874c01598428c7b70416692f6899c2b22d735154. Could you try the lastest `master` and check whether the issue persists?

**Comment by wbwqq** [comment]
The site wouldn't build with the latest master. I obtain the following error:

```
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: D:\projects\python2\site
ERROR   -  Error building page 'index.md': partials/language/.html
Traceback (most recent call last):
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\willi\AppData\Local\Programs\Python\Python38-32\Scripts\mkdocs.exe\__main__.py", line 7, in <module>
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\mkdocs\__main__.py", line 152, in build_command
    build.build(config.load_config(**kwargs), dirty=not clean)
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\mkdocs\commands\build.py", line 292, in build
in _build_page
    output = template.render(context)
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\projects\python2\mkdocs-material/material\main.html", line 4, in top-level template code
    {% extends "base.html" %}
  File "D:\projects\python2\mkdocs-material/material\base.html", line 4, in top-level template code
 make_module
    return TemplateModule(self, self.new_context(vars, shared, locals))
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\environment.py", line 1238, in __init__
    body_stream = list(template.root_render_func(context))
  File "D:\projects\python2\mkdocs-material/material\partials\language.html", line 4, in top-level template code
    {% import "partials/language/" ~ config.theme.language ~ ".html" as lang %}
  File "c:\users\willi\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\loaders.py", line 197, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: partials/language/.html
```

**Comment by squidfunk** [comment]
That is weird because the GitHub workflow which builds and deploy the site worked:
https://github.com/squidfunk/mkdocs-material/actions/runs/364605820

**Comment by squidfunk** [comment]
Okay, I think I know what might be happening. When you clone from GitHub, MkDocs will ignore the `mkdocs_theme.yml`. For this reason, you need to add the following to your `mkdocs.yml`:

``` yaml
theme:
  ...
  language: en
  palette:
    scheme: default
    primary: indigo
    accent: indigo
  font:
    text: Roboto
    code: Roboto Mono
  favicon: assets/favicon.png
```

**Comment by squidfunk** [comment]
Side note: I updated the documentation in e31e63fe and added the [necessary configuration values](https://squidfunk.github.io/mkdocs-material/creating-your-site/#minimal-configuration) when installing from `git`. It wasn't very clear before.

**Comment by squidfunk** [comment]
@wbwqq have you managed to try the latest master?

**Comment by squidfunk** [comment]
Okay... so I've powered up a virtual machine and tested in Edge, which shows no scrollbar. Unfortunately, I don't have the time to install and test Firefox, but I guess it's rather safe to assume that it should be fixed as well, as scrollbars are system-level.

![Bildschirmfoto 2020-11-20 um 08 52 14](https://user-images.githubusercontent.com/932156/99774393-02b6c100-2b0e-11eb-98db-1febadce7d1e.png)

I'll issue a release within the next days.

**Comment by squidfunk** [comment]
Released as part of 6.1.6
